.changeset/yellow-numbers-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Added missing `.eslintignore` file
```

Comment:
You may want to add a line here telling people who already have a repo, what they should do locally.